python/social_bot/tasks.py

```diff
@@ -238,10 +246,9 @@ def __init__(self,
             max_reward_q_length (int): how many recent rewards to consider when estimating agent accuracy.
             reward_weight (float): the weight of the reward, is used in multi-task case
             move_goal_during_episode (bool): if True, the goal will be moved during episode, when it has been achieved
-            end_episode_after_success (bool): if True, the episode will end once the goal is reached. A True value of this
-                flag will overwrite the effects of flags ``switch_goal_within_episode`` and ``move_goal_during_episode``.
             success_with_angle_requirement: if True then calculate the reward considering the angular requirement
             additional_observation_list: a list of additonal objects to be added
+            use_full_states (bool): For non-image observation, use full states of the world
```

Comment:
task_specific_observation() is supposed for non-image case and use full states of the world by default.  What is the difference use_full_states or not?